<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>supplies</class>
 <widget class="QMainWindow" name="supplies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1414</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчет по книгам и авторам</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(120, 180,220);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnMenu">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 112,230);
</string>
    </property>
    <property name="text">
     <string>Меню</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twotchet">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>100</y>
      <width>1401</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(120, 180,220);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbID_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Отчет по книгам и авторам&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectAll">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 112,230);
</string>
    </property>
    <property name="text">
     <string>Вывести отчет</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teResult">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32,120, 215);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSea">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 112,230);
</string>
    </property>
    <property name="text">
     <string>Поиск по фамилии</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minCount">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32,120, 215);
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxCount">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32,120, 215);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectAll_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 112,230);
</string>
    </property>
    <property name="text">
     <string>Фильтрировать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Мин_кол</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс_кол</string>
    </property>
   </widget>
   <zorder>teResult</zorder>
   <zorder>btnMenu</zorder>
   <zorder>twotchet</zorder>
   <zorder>lbID_2</zorder>
   <zorder>btnSelectAll</zorder>
   <zorder>leSea</zorder>
   <zorder>btnSea</zorder>
   <zorder>minCount</zorder>
   <zorder>maxCount</zorder>
   <zorder>btnSelectAll_2</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1414</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
